apps/web/src/components/auth/LoginForm.tsx

```diff
@@ -108,19 +113,21 @@ export function LoginForm({}: Props) {
           placeholder="Type your password..."
           data-test-id="password"
         />
-        <Link to="/auth/reset/request">
-          <Text my={30} gradient align="center">
-            Forgot Your Password?
-          </Text>
-        </Link>
+        {!isFromVercel && (
+          <Link to="/auth/reset/request">
+            <Text my={30} gradient align="center">
+              Forgot Your Password?
+            </Text>
+          </Link>
+        )}
         <Button mt={60} inherit loading={isLoading} submit data-test-id="submit-btn">
           Sign In
         </Button>
         <Center mt={20}>
           <Text mr={10} size="md" color={colors.B60}>
             Don't have an account yet?
           </Text>
-          <Link to="/auth/signup">
+          <Link to={isFromVercel ? `/auth/signup?code=${code}&next=${next}` : '/auth/signup'}>
```

Comment:
This should be made a const instead to keep logic and html seperated -> `const signupLink = isFromVercel ? /auth/signup?code=${code}&next=${next} : '/auth/signup'`